### ISSUE TITLE
Genetic codes are out of date

### ISSUE THREAD

**Issue by Benjamin-Lee** [scikit-bio/scikit-bio#1659]
It now goes up to 33! (according to the [NCBI table](https://www.ncbi.nlm.nih.gov/Taxonomy/Utils/wprintgc.cg))

**Comment by exTerEX** [comment]
I'm looking into this issue. It's unproblematic to implement code 26, 29, 30 and 33 from [NCBI table](http://www.ncbi.nlm.nih.gov/Taxonomy/Utils/wprintgc.cgi), but I'm unsure how to handle code 27, 28 and 31 because the genetic code include codons encoding for either an amino acid or a stop codon, depending on its position on the mRNA strand. This context-dependent translation of codons is described in [Swart et al., 2016](https://www.doi.org/10.1016/j.cell.2016.06.020). Are there any desire to see this implemented? Or else I think it would be best to not add code 27, 28 and 31, but proceed to add the other NCBI-codes.

**Comment by Benjamin-Lee** [comment]
I would suggest to skip it since context-dependent translation is out of scope for this project. Maybe raise an error explaining the reason why?

**Comment by exTerEX** [comment]
I agree. Checking if anybody tries to access code 27, 28 or 31 from `GeneticCode().from_ncbi()` and raise a `ValueError` with a short explanation and/or reference to this issue.
> I would suggest to skip it since context-dependent translation is out of scope for this project. Maybe raise an error explaining the reason why?

**Comment by FormulaCarbon** [comment]
is this still supp0sed to be open?
